lib/core.js

```diff
@@ -1022,6 +1029,8 @@ function addReorgHint (list, tree, batch) {
 }
 
 function updateUserData (list, key, value) {
+  value = unslab(value)
```

Comment:
defer this to after the validation

Reply:
You mean deferring to when `value` is actually assigned to something (lines 1036 and 1041)? 
(not sure where validation happens)

Reply:
ah i misremembered, this is ok